logger.go

```diff
@@ -0,0 +1,25 @@
+package exasol
+
+import (
+	"errors"
+	"log"
+	"os"
+)
+
+var errorLogger = Logger(log.New(os.Stderr, "[exasol] ", log.LstdFlags|log.Lshortfile))
+
+// Logger is used to log critical error messages.
```

Comment:
If this interface is only used to log errors, maybe it's more precise to name it ErrorLogger?

Reply:
I would keep it `Logger` if we want to use it for normal log messages in the future

Reply:
Okay. Then it's fine to merge